src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Extensions/ClaimsPrincipalExtensions.cs

```diff
@@ -0,0 +1,99 @@
+using System.Security.Claims;
+using AltinnCore.Authentication.Constants;
+
+namespace Altinn.Platform.Authenticaiton.Extensions
+{ 
+    /// <summary>
+    /// Helper methods to extend ClaimsPrincipal. 
+    /// </summary>
+    public static class ClaimsPrincipalExtensions
+    {
+        /// <summary>
+        /// Gets the userId or the orgNumber or null if neither claims are present.
+        /// </summary>
+        public static string GetUserOrOrgId(this ClaimsPrincipal user)
+        {
+            int? userId = GetUserIdAsInt(user);
+            if (userId.HasValue)
+            {
+                return userId.Value.ToString();
+            }
+
+            int? orgId = GetOrgNumber(user);
+            if (orgId.HasValue)
+            {
+                return orgId.Value.ToString();
+            }
+
+            return null;
+        }
+
+        /// <summary>
+        /// Get the org identifier string or null if it is not an org.
+        /// </summary>        
+        public static string GetOrg(this ClaimsPrincipal user)
+        {
+            if (user.HasClaim(c => c.Type == AltinnCoreClaimTypes.Org))
+            {
+                Claim orgClaim = user.FindFirst(c => c.Type == AltinnCoreClaimTypes.Org);
+                if (orgClaim != null)
+                {
+                    return orgClaim.Value;
+                }
+            }
+
+            return null;
+        }
+
+        /// <summary>
+        /// Returns the organisation number of an org user or null if claim does not exist.
+        /// </summary>
+        public static int? GetOrgNumber(this ClaimsPrincipal user)
```

Comment:
I think, organization number should always be a string. In which cases can it be an int?